client/src/components/InteractionCard/InteractionCard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import "./InteractionCard.css";
+import { BrowserRouter as Router, Link, Route, Switch } from "react-router-dom";
+import Button from "../Button/Button";
+import ProfilePicture from "../ProfilePicture/ProfilePicture";
+import Image from "../../../../assets/images/profile_large.png";
+// Function based React Component
+
+const InteractionCard = props => {
+  // Default Class to apply to Component
+  return (
+    <div className="InteractionCard">
+      <ProfilePicture to="#" image={Image} />
```

Comment:
@karenheyn This should be to the `to` prop that's passed in (i.e. `to={props.to}`)